hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -240,10 +253,9 @@ protected void loadConf(Configuration conf) {
     curFunctionCosts = new double[costFunctions.size()];
     tempFunctionCosts = new double[costFunctions.size()];
 
-    LOG.info("Loaded config; maxSteps=" + maxSteps + ", runMaxSteps=" + runMaxSteps,
-      ", stepsPerRegion=" + stepsPerRegion +
-      ", maxRunningTime=" + maxRunningTime + ", isByTable=" + isByTable + ", CostFunctions=" +
-      Arrays.toString(getCostFunctionNames()) + " etc.");
+    LOG.info("Loaded config; maxSteps={}, runMaxSteps={}, stepsPerRegion={}, maxRunningTime={}, "
```

Comment:
Since this is slf4j, this would only print out `Loaded config; maxSteps=1000000, runMaxSteps=false` and drop the rest. This is because these extra args are considered format arguments, but there are no variables in the format string. I updated this to the proper slf4j usage so the rest of the intended details could be printed.

Reply:
Nice clean up.